packages/commercetools/theme/pages/Checkout.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div id="checkout">
+    <div class="checkout">
+      <div class="checkout__main">
+        <SfSteps :active="currentStepIndex" @change="handleStepClick" v-if="!isThankYou" :class="{ 'checkout__steps': true, 'checkout__steps-auth': isAuthenticated }">
+          <SfStep v-for="(step, key) in STEPS" :key="key" :name="step">
+            <nuxt-child />
+          </SfStep>
+        </SfSteps>
+        <nuxt-child v-else  />
+      </div>
+      <div class="checkout__aside desktop-only" v-if="!isThankYou">
+        <transition name="fade">
+          <CartPreview v-if="showCartPreview" key="order-summary" />
+          <OrderReview v-else key="order-review" />
+        </transition>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
```

Comment:
I've seen in a couple of files `lang="ts"`. Could you add it here as well?

Reply:
we have to add support for TS everywhere in vue files, but not now, once we clean up everything we will apply TS support in the theme as well